torch/cuda/__init__.py

```diff
@@ -316,9 +316,10 @@ def stream(stream):
         match the stream.
     """
     if stream is None:
-        yield
+    #    yield
         return
-    src_prev_stream = current_stream()
+    #src_prev_stream = current_stream()
+    src_prev_stream = torch.cuda_getCUrrentStream(0)
```

Comment:
```suggestion
    src_prev_stream = torch.cuda_getCurrentStream(0)
```

Reply:
Oops. Yeah. I had fixed this. Not sure how this came on to the PR. I'll fix this.